Mobile Buy SDK/Mobile Buy SDK Tests/BUYApplePayTokenTests.m

```diff
@@ -0,0 +1,71 @@
+//
+//  BUYApplePayTokenTests.m
+//  Mobile Buy SDK
+//
+//  Created by Dima Bart on 2016-05-12.
+//  Copyright © 2016 Shopify Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import <PassKit/PassKit.h>
```

Comment:
Perhaps, but we don't use them for any other modules like `Foundation` or `UIKit` or `XCTest`. I guess I just prefer the consistency.